tests/testthat/test-h_bracket.R

```diff
@@ -0,0 +1,20 @@
+
+
+
+test_that("h_bracket works as expected", {
+    st_mod <- StanModule(
+        functions = "string",
+        data = "string",
+        parameters = "string",
+        transformed_parameters = "string",
+        generated_quantities = "string",
+        priors = list("item1;", "item2;")
```

Comment:
Given the separation of `priors` and `model` now does it make sense to have unnamed priors anymore ? Perhaps we should throw an error for this ?

Reply:
Definitely, the priors ought to have names. I will make another test for the existence of the names of the priors.

Reply:
Sorry did you add any assertions for this ?

Reply:
I decided to ensure that priors have names with `assert_that` during the creation of the object in my todays commit .